test/e2e/simple.go

```diff
@@ -31,7 +31,8 @@ func E2ESimple(logger *log.Logger) error {
 	logger.Println("Creating txsim")
 	endpoints, err := testNet.RemoteGRPCEndpoints()
 	testnet.NoError("failed to get remote gRPC endpoints", err)
-	err = testNet.CreateTxClient("txsim", testnet.TxsimVersion, 1, "100-2000", 100, testnet.DefaultResources, endpoints[0])
+	err = testNet.CreateTxClient("txsim", testnet.TxsimVersion, 10,
```

Comment:
[To reviewers] The sequence count has been increased to 10 to address a flaky behavior in the E2ESimple test. This change ensures there are sufficient transactions at the end of the experiment.